src/components/PreferencesPage.vue

```diff
@@ -126,6 +126,22 @@
             @change="onChange($event)"
         />
     </label>
+    <!-- chapters layout on mobile -->
+    <label class="pref" for="chkMinimizeChapters">
```

Comment:
I guess the pref should only be visible on mobile devices. You can just hide it using sth like `style="md:hidden"` ( not tested)

Reply:
done